docs/blog/2019-11-22-getting-started-with-gatsby-cloud/index.md

```diff
@@ -0,0 +1,83 @@
+Here's the earliest version we shipped.
+
+![A screenshot of an earlier onboarding design](first-phase.png)
+
+### Next phase
+Here's a screenshot of a Figma prototype we tested based on feedback on the first version.
+
+![A screenshot of a prototype that we tested next](middle-option.png)
+
+### We’re prioritizing what we build according to user needs
+
+From this process of testing prototypes, implementing changes, and testing again, we could answer these questions:
+- What blocks people from onboarding onto Gatsby Cloud?
+- What creates friction for people onboarding onto Gatsby Cloud?
+- What creates value for people onboarding onto Gatsby Cloud?
+
+## What blocks people from onboarding onto Gatsby Cloud?
+
+Connecting with a GitHub organization is the most confusing step, for several reasons. First of all, it’s not obvious in the product that we only offer GitHub integrations at this time (GitLab and Bitbucket will come eventually), so many users click around, looking for a way to connect to GitLab and/or Bitbucket.
+
+We’re currently shipping some more improvements to the GitHub integration that I’ll blog about in a separate post soon!
+
+## What created friction for people onboarding onto Gatsby Cloud?
+Choosing a CMS was confusing to four of the eight testers. None of them had an opinion on which CMS to choose. We learned from this that an MDX starter would avoid requiring users to choose a CMS. 
+
+## What creates value for people onboarding onto Gatsby Cloud?
+There were some good things about onboarding, and we want those good things to persist and grow, if possible.
+Choosing a starter was fun, and 3 of the users wanted more details about the starters. On starter cards, we added a link to their github repos or /starters/ page on .org
+
+### Product, design, and engineering are doing something about the feedback
+
+We’re tracking all feedback in GitHub and shipping improvements according to a prioritization system that relies on three metrics, thanks to [Step 3 of Barron Caster’s 3x Conversion Playbook](https://medium.com/hackernoon/the-3x-conversion-playbook-3a9b954b937d).
+- How confident are we that this problem is real?
+- How much of an impact would fixing this problem have on major user workflows?
+- How quickly can we fix this problem?
+
+Product, the design team and the Cloud engineering team are collaboratively designing solutions. And then...yep, you guessed. Shipping and testing again.
+
+## Current onboarding
+Here is a screenshot of the current onboarding experience and a brief overview of some planned improvements.
+
+![A screenshot of the current onboarding experience](final-state-comment.png)
+
+## Planned improvements to Gatsby Cloud
+We’ve shipped a ton of improvements already, and have a few more that are in progress as we speak!
+- Adding an MDX starter
+- Clarifying the step in which users add a github organization because using a starter vs using an existing Gatsby site require different flows
```

Comment:
```suggestion
- Clarifying the step in which users add a GitHub organization because using a starter vs using an existing Gatsby site require different flows
```